ui/app/src/utils/i18n-legacy.ts

```diff
@@ -1478,3 +1478,15 @@ export const siteSuccessMessages = defineMessages({
     defaultMessage: 'Site deleted successfully'
   }
 });
+
+export const sitePolicyMessages = defineMessages({
+  itemPastePolicyConfirm: {
+    id: 'pastePolicy.confirm',
+    defaultMessage:
+      'The selected {action} target goes against site policies for the destination directory. • Original path: "{path}", • Suggested path is: "{modifiedPath}". Would you like to use the suggested path?'
+  },
+  itemPastePolicyError: {
+    id: 'pastePolicy.error',
+    defaultMessage: 'The selected copy|cut target goes against site policies for the destination directory.'
+  }
+});
```

Comment:
Is this being used on legacy? Why is this here?

Reply:
to avoid creating it on Epic, will move to the epic